Dockerfile

```diff
@@ -32,14 +34,21 @@ COPY scripts/bootstrap.py /usr/local/jenkins/bin/bootstrap.py
 COPY conf/jenkins/config.xml "${JENKINS_STAGING}/config.xml"
 COPY conf/jenkins/jenkins.model.JenkinsLocationConfiguration.xml "${JENKINS_STAGING}/jenkins.model.JenkinsLocationConfiguration.xml"
 COPY conf/jenkins/nodeMonitors.xml "${JENKINS_STAGING}/nodeMonitors.xml"
-COPY conf/tomcat/server.xml "${CATALINA_HOME}/conf/server.xml"
-COPY conf/tomcat/Catalina/localhost/rewrite.config "${CATALINA_HOME}/conf/Catalina/localhost/rewrite.config"
 
-RUN apt-get update
-RUN apt-get install -y git python zip jq
 RUN /usr/local/jenkins/bin/plugin_install.sh "${JENKINS_STAGING}/plugins"
 
 # Override the default property for DNS lookup caching
 RUN echo 'networkaddress.cache.ttl=60' >> ${JAVA_HOME}/jre/lib/security/java.security
 
-CMD /usr/local/jenkins/bin/bootstrap.py && catalina.sh run
+CMD /usr/local/jenkins/bin/bootstrap.py && nginx && \
+java -Dhudson.udp=-1                                \
+    -Djava.awt.headless=true                        \
+    -DhudsonDNSMultiCast.disabled=true              \
+    -jar ${JENKINS_WAR}                             \
+    --httpPort=8080                                 \
+    --webroot=JENKINS_WAR                           \
+    --ajp13Port=-1                                  \
+    --httpListenAddress=0.0.0.0                     \
+    --ajp13ListenAddress=127.0.0.1                  \
+    --preferredClassLoader=java.net.URLClassLoader  \
+    --prefix=/service/jenkins
```

Comment:
The value for `--prefix` should be based on the environment variable `$JENKINS_CONTEXT`